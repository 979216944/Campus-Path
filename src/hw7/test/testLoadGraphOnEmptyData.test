# Test load a graph with no nodes.

# Load graph
LoadGraph empty empty.tsv

# List nodes
ListNodes empty

# Find a nonexist minimum-cost path
FindPath empty node1 node2