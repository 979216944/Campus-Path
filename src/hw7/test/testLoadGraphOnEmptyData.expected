# Test load a graph with no nodes.

# Load graph
loaded graph empty

# List nodes
empty contains:

# Find a nonexist minimum-cost path
unknown character node1
unknown character node2